{
    "checksum": "309dd820e8649ea421297180e8b346bfaf6da14fa96f883ae7e75a21c90f1cf1",
    "file": "sessionmanagerexception.py",
    "mime": {
        "encoding": "us-ascii",
        "type": "text/x-python"
    },
    "number_of_duplicates": 2,
    "package": "python-omemo",
    "path": "python-omemo/0.10.3-1.1/omemo/exceptions/sessionmanagerexception.py",
    "pkg_infos": {
        "area": "main",
        "copyright": true,
        "ctags_count": 0,
        "license": "/copyright/license/python-omemo/0.10.3-1.1/",
        "metric": {
            "size": 296
        },
        "pts_link": "https://tracker.debian.org/pkg/python-omemo",
        "sloc": [
            [
                "python",
                1626
            ],
            [
                "makefile",
                3
            ]
        ],
        "suites": [
            "bookworm",
            "bullseye",
            "sid"
        ],
        "vcs_browser": "https://salsa.debian.org/xmpp-team/python-omemo",
        "vcs_type": "git"
    },
    "raw_url": "/data/main/p/python-omemo/0.10.3-1.1/omemo/exceptions/sessionmanagerexception.py",
    "stat": {
        "perms": "rw-r--r--",
        "size": 100,
        "symlink_dest": null,
        "type": "-"
    },
    "text_file": true,
    "type": "file",
    "version": "0.10.3-1.1"
}